hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1147,11 +1147,17 @@ private function updateJobContractHealthSchema() {
    */
   private function updateJobContractPensionSchema() {
     $pensionTableName = CRM_Hrjobcontract_BAO_HRJobPension::getTableName();
-    CRM_Core_DAO::executeQuery("
-      ALTER TABLE $pensionTableName
-      MODIFY pension_type VARCHAR(512),
-      DROP FOREIGN KEY  pension_type_contact_id_fk
-    ");
+    $pensionTypeConstraintExist =  CRM_Core_DAO::checkConstraintExists(
+      $pensionTableName,
+      'pension_type_contact_id_fk'
```

Comment:
You can extract this `pension_type_contact_id_fk` into a variable since you are using the string in more than one place